HandPuzzle/Handpuzzelcode/Handpuzzelcode.ino

```diff
@@ -0,0 +1,23 @@
+int analogPin = A3; // potentiometer wiper (middle terminal) connected to analog pin 3
+                    // outside leads to ground and +5V
+int val = 0;  // variable to store the value read
```

Comment:
Ik zou hier dan readValue van maken, dan is het direct duidelijk wat er met de variable bedoelt wordt zelfs zonder commentaar.